### PR TITLE
[CustomFieldsColumns] extraction of custom fields implemented #79

### PR DESCRIPTION
**Pull request by krzysztofruszczynski**
#79

### CODE REVIEW

File: src/Form/ColumnsExtractor.php [hunk 1]
```diff
@@ -9,6 +9,11 @@
  */
 class ColumnsExtractor
```
Comment (markussc) [inline src/Form/ColumnsExtractor.php]: Please add an explanation regarding the purpose of the class (where do we extract custom fields from etc.)

File: src/Form/ColumnsExtractor.php [hunk 2]
```diff
@@ -33,6 +38,32 @@ public function getColumnLabel($formElement)
...
+    }
+
+    /**
+     * Method can be overwritten in order to make custom style for columns.
+     *
+     * @param \Symfony\Component\Form\Form $formElement
+     *
+     * @return string style for column
+     */
+    public function getColumnStyle($formElement)
```
Comment (markussc) [inline src/Form/ColumnsExtractor.php]: Maybe add type hint?

File: src/Form/ColumnsExtractor.php [hunk 3]
```diff
@@ -56,4 +87,38 @@ public function getColumns($form)
...
+    /**
+     * Method adding custom fields labels to given array with column names.
+     * Method counts number of custom fields (result available via getCustomFieldsNumberOfColumns method).
+     *
+     * @param \Symfony\Component\Form\AbstractType $form    form, from which custom fields are extracted
+     * @param array                                $columns current array with column names (can be empty, then return array contains only custom fields)
+     *
+     * @return array array from input with custom fields attached
+     */
+    public function addCustomFieldsColumns($form, $columns = array())
```
Comment (markussc) [inline src/Form/ColumnsExtractor.php]: maybe add type hint for $form ?

File: src/Form/ColumnsExtractor.php (hunk 1 above)
Comment (krzysztofruszczynski) [reply src/Form/ColumnsExtractor.php]: added

File: src/Form/ColumnsExtractor.php (hunk 2 above)
Comment (krzysztofruszczynski) [reply src/Form/ColumnsExtractor.php]: as discussed, we can leave it without type hinting for more flexibility

File: src/Form/ColumnsExtractor.php (hunk 3 above)
Comment (krzysztofruszczynski) [reply src/Form/ColumnsExtractor.php]: as discussed, we can leave it without type hinting for more flexibility

**Pull request merged**